### ISSUE TITLE
go2xunit is creating a lot of build failures

### ISSUE THREAD

**Issue by bjornick** [vanadium/issues#982]
While trying to make the release candidate this morning, the go linux tests failed 3 consecutive times due to failures in go2xunit.

https://veyron.corp.google.com/jenkins/job/release-candidate-2-test/ARCH=amd64,OS=linux,TEST=vanadium-go-test/479/console
https://veyron.corp.google.com/jenkins/job/release-candidate-2-test/ARCH=amd64,OS=linux,TEST=vanadium-go-test/480/console
https://veyron.corp.google.com/jenkins/job/release-candidate-2-test/ARCH=amd64,OS=linux,TEST=vanadium-go-test/481/console

Why are we using this library again?

**Comment by cosnicolaou** [comment]
it's failing because the test output contains a failure:

fail v.io/x/ref/lib/security/securityflag
04:24:27 === RUN   TestNewAuthorizerOrDie

we're using it to convert between go's undefined test output messages and the xunit format expected by jenkins and other tools. The orphan message occurs because there's RUN line but no corresponding FAIL because the test failed (most likely a panic) without outputing. Even if we wrote our own parser, we'd have to deal with the same problem/error.

In short, the test is actually failing, but go2xunit's error message is not clear that that's the case.
